main.js

```diff
@@ -1,26 +1,80 @@
-import oDate from './foundation/date';
-// TODO: integrate n-layout into this repo
-import layout from 'n-layout';
 // TODO: integrate next-js-setup into this repo
 import setup from 'next-js-setup';
-// TODO: these should be configurable as they're not foundations
-import prompts from './components/message-prompts';
+import date from './foundation/date';
+import header from './components/header';
+import cookieMessage from './components/cookie-message';
+import welcomeMessage from './components/welcome-message';
+import messagePrompts from './components/message-prompts';
 import { client as myFtClient, ui as myFtUi } from './components/myft';
 
+const presets = {
+	discrete: {
+		header: true,
+		date: true
+	},
+	all: {
+		header: true,
+		date: true,
+		cookieMessage: true,
+		welcomeMessage: true,
+		myft: true,
+		messagePrompts: true
+	}
+}
+
 module.exports = {
-	bootstrap: function (cb, opts = {}) {
+	bootstrap: function (opts = {}, cb) {
+
+		if (opts.preset) {
+			opts = Object.assign(presets[opts.preset], opts);
```

Comment:
So `bootstrap({ preset: 'all', myft: false }`, nice. Maybe options should always be applied though, E.G.:

```js
opts = Object.assign({}, opts.preset ? presets[opts.preset] : null, opts)
```